src/Calculator/App.xaml.cpp

```diff
@@ -198,12 +198,12 @@ void App::OnLaunched(LaunchActivatedEventArgs ^ args)
         // If the app got pre-launch activated, then save that state in a flag
         m_preLaunched = true;
     }
+    NavCategory::InitializeCategoryManifest(args->User);
```

Comment:
Any scenarios where this would be null? And if so, does it make sense to fallback to the User::FindAllAsync logic?

Reply:
Since `User::FindAllAsync` does not guarantee that the returned user is the currently active user, fundamentally its result is not super meaningful. So, if `args->User` is null, it'd make more sense to go along with the comment below, which would the permission to true.